source/features/discussion-links-on-repo-lists.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import issueIcon from 'octicon/issue-opened.svg';
+import {isUserProfileRepoTab} from '../libs/page-detect';
+import pullRequestIcon from 'octicon/git-pull-request.svg';
+import observeElement from '../libs/simplified-element-observer';
+
+function addIcons(): void {
+	for (const repository of select.all('[itemprop="owns"]:not(.rgh-discussion-links), .repo-list-item')) {
```

Comment:
Can you select the repository link/name and then work back from there? A `repositoryLink.closest('x')` would probably end up being slightly safer/shorter

Reply:
```suggestion
	const repositories = select.all([
			'[itemprop="name codeRepository"]', // `isUserProfileRepoTab`
			'[data-hydro-click*=\'"model_name":"Repository"\']' // `isGlobalSearchResults`
	]);
	for (const repository of repositories) {
```